### PR TITLE
Added pep8 check on pull request pushes

### PR DESCRIPTION
**Pull request by DavidBergevoet**

### CODE REVIEW

File: .github/workflows/pep8-check.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+jobs:
+  pep8_check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: pip install autopep8
```
Comment (InzeNL) [inline .github/workflows/pep8-check.yml:16]: Looking at https://github.com/DavidBergevoet/Setup-Reservation/blob/main/.github/workflows/python-app.yml, pip packages are being cached. It seems that `actions/setup-python` is not used here, so I'm not 100% sure if that is possible here. Could you check?

File: requirements.txt [hunk 2]
```diff
@@ -16,3 +16,4 @@ tzdata==2024.1
 tzlocal==5.2
 Werkzeug==3.0.3
 WTForms==3.1.2
+autopep8==2.3.1
```
Comment (InzeNL) [inline requirements.txt:19]: This isn't required by the application, is it? Perhaps it makes more sense to add the installation in the `README.md`, but don't require it for the application itself

**Comment by InzeNL** [comment]
Is this something we would cover in the contribution guidelines? Requiring people to use pep8?

**Review by InzeNL — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 2 above)
Comment (DavidBergevoet) [reply requirements.txt:19]: Removed from requirements and added installation steps to the README.md

File: .github/workflows/pep8-check.yml (hunk 1 above)
Comment (DavidBergevoet) [reply .github/workflows/pep8-check.yml:24]: I changed it so this will also use the setup-python, is this what you meant?

**Review by InzeNL — APPROVED** [review]

**Pull request merged**